test/e2e/awsiam_test.go

```diff
@@ -109,6 +110,28 @@ func TestVSphereKubernetes121BottleRocketAWSIamAuth(t *testing.T) {
 	runAWSIamAuthFlow(test)
 }
 
+func TestCloudStackKubernetes120AWSIamAuth(t *testing.T) {
```

Comment:
Let's not include these special tests just yet, unless we anticipate these features being used right away. This may cause a lot of load in the short term until we consolidate some of these tests.